lib/tizen_sdk.dart

```diff
@@ -236,17 +236,31 @@ class Rootstrap {
   bool get isValid => rootDirectory.existsSync();
 }
 
-/// Converts [targetArch] to an arch name that corresponds to the `BUILD_ARCH`
+/// Converts [arch] to an arch name that corresponds to the `BUILD_ARCH`
 /// value used by the Tizen native builder.
-String getTizenBuildArch(String targetArch) {
-  switch (targetArch) {
+String getTizenBuildArch(String arch) {
+  switch (arch) {
     case 'arm':
       return 'armel';
     case 'arm64':
       return 'aarch64';
     case 'x86':
       return 'i586';
     default:
-      return targetArch;
+      return arch;
+  }
+}
+
+/// Converts [arch] to an arch name that the Tizen CLI tool expects.
+String getTizenCliArch(String arch) {
+  switch (arch) {
+    case 'arm':
+      return 'arm';
+    case 'arm64':
```

Comment:
Is the conversion necessary only for``` arm64```?

Reply:
Yes, I made https://github.com/flutter-tizen/flutter-tizen/pull/175/files#diff-32efabcee487e07581c54bc0c3ac9e16b5401ebd7b15ad5337b96186fe36b41cL780-L787 more explicit.